### PR TITLE
TextV1 migration for win32

### PR DESCRIPTION
**Pull request by lenahong**
### Platforms Impacted
- [ ] iOS
- [ ] macOS
- [ ] win32 (Office)
- [ ] windows
- [ ] android

### Description of changes
Moving files under experimental text into regular text package.
Removed jest.config from experimental-text, as experimental-text now pull from the migrated text.
Also moving the old text under deprecated.

Removing other components' dependencies on experimental-text, and changing it to TextV1.

TODO: Spec and migration documentations will be added separately.

### Verification
No changes seen in test, everything renders as expected.
No build errors.

### Pull request checklist

This PR has considered (when applicable):
- [X] Automated Tests
- [ ] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

File: apps/component-generator/component-templates/ComponentTemplate/package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     "directory": "packages/components/ComponentTemplate"
   },
   "dependencies": {
-    "@fluentui-react-native/experimental-text": ">=0.7.17 <1.0.0",
+    "@fluentui-react-native/text": ">=0.12.1 <1.0.0",
```
Comment (rurikoaraki) [inline apps/component-generator/component-templates/ComponentTemplate/package.json:26]: nit: alphabetical please

File: apps/fluent-tester/src/testPages.ts [hunk 2]
```diff
@@ -95,11 +94,6 @@ export const tests: TestDescription[] = [
     component: TextTest,
     testPage: HOMEPAGE_TEXT_BUTTON,
   },
-  {
```
Comment (rurikoaraki) [inline apps/fluent-tester/src/testPages.ts:98]: I don't see the experimetnal text test page contents moving anywhere, so is removing the experimental test page from the list intended?

Comment (lenahong) [reply apps/fluent-tester/src/testPages.ts:98]: Ah I was going to but I decided to keep the test pages as they are. I'll put them back in!

**Review by rurikoaraki — APPROVED** [review]
Just one nit but otherwise looks good

**Comment by Saadnajmi** [comment]
Since macOS has e2e testing now... 

/azp run

**Review by ValentinaKozlova — APPROVED** [review]

**Pull request merged**